python/shotgun_desktop/initialization/install.py

```diff
@@ -27,17 +27,26 @@
 from .. import utils
 from . import constants
 from . import shotgun
+from ..logger import get_logger
 
 
 class InstallThread(QtCore.QThread):
-    stepDone = QtCore.Signal(str)
```

Comment:
Signaling between threads implies having a active message loop, which we don't have. This looks like something that was started but never completed. Anyhow, the new system is simpler and works without a message loop. Also, everything is still logged so the user experience is the same.